static/js/marathon.js

```diff
@@ -34,6 +34,7 @@ let app = new Vue({
         startArticle: "",    //For all game modes, this is the first article to load
         lastArticle: "",
         currentArticle: "",
+        articlePreview: "",
```

Comment:
initializing it here as an empty string is rather confusing. I think either use `null` or expand out the expected fields like so
```
articlePreview: {
    image: null,
    html: null
}
```

Reply:
Good point, will use `null`